Bug 1119544 - Remove the unnecessary l10n code from FMD

lissyx opened the pull request.

Commit:
Bug 1119544 - Remove the unnecessary l10n code from FMD

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/1645737?v=3" alt="lissyx" width="50px" height="50px" />  (lissyx)](https://github.com/lissyx) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ffbf06c6b7e2)

lissyx committed:
Bug 1119544 - Fix FindMyDevice test

We were lacking:
 - proper stubbing of mozId API, specifically no watch()
 - Config object was missing

Because previously, the l10n loading code was used to inhibit triggering
of FMD.init() since we do not really needed it.

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/1645737?v=3" alt="lissyx" width="50px" height="50px" />  (lissyx)](https://github.com/lissyx) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=461fd0a4ef38)

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/1645737?v=3" alt="lissyx" width="50px" height="50px" />  (lissyx)](https://github.com/lissyx) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8d7c952169e5)
